helmfile/upstream/thanos/values.yaml

```diff
@@ -3327,6 +3752,9 @@ receiveDistributor:
   ## @param receiveDistributor.replicaLabel Label to treat as a replica indicator along which data is de-duplicated
   ##
   replicaLabel: replica
+  ## @param receiveDistributor.replicationFactor Thanos Receive Distributor replication-factor
+  ##
+  replicationFactor: 1
```

Comment:
You should change in the release templates so we also configure this here.

Reply:
I think i got what you meant 😅 see [this](https://github.com/elastisys/compliantkubernetes-apps/pull/1600/commits/135c03d11b37b9c8f3752a8386f23dc8852dec41) commit.